model/review.ts

```diff
@@ -0,0 +1,39 @@
+import {model, Schema} from "mongoose";
+import {ProductInterface} from "./product";
+
+export interface ReviewInterface {
```

Comment:
It should also have a title field. and if we have a date field, why not use Date type?